File: f77/build06/setval.f
C
C     Set values common block variables and declarations
C
      Integer MAX_NUMBER_OF_REAL_VALUES,
     &     Number_Of_Real_Values,
     &     Set_Real_Value_Index(50)
       Real Set_REal_Value(50)
       Common /Set_Value_Com/ Number_Of_Real_Values,
     &      Set_Real_Value,
     &      Set_Real_Value_Index
       Parameter [MAX_NUMBER_OF_REAL_VALUES = 50)
C

File: compcert/driver/setval.c
#include "../driver/setval.h"

// Current number of real variables to be initialized [Integer]
INT8 number_of_real_values;

// Real_Array locations for initial values [Integer Array]
INT8 set_real_value_index[MAX_NUMBER_OF_REAL_VALUES];

// Initial values for real variables [Real Array]
CELL set_real_value[MAX_NUMBER_OF_REAL_VALUES];
